PowerShellTools.HostService/ServiceManagement/Debugging/PowershellDebuggingServiceUtilities.cs

```diff
@@ -135,6 +145,16 @@ private void DebuggerFinished()
             }
         }
 
+        private bool IsDebuggerActive(System.Management.Automation.Debugger debugger)
+        {
+            if (_installedPowerShellVersion >= RequiredPowerShellVersionForRemoteSessionDebugging)
+            {
+                return debugger.IsActive;
+            }
+
+            return false;

```

Comment:
Let's put a short comment here about why we are returning false here for v4- versions.